lib/menu.rb

```diff
@@ -0,0 +1,31 @@
+class Menu
+
+  def initialize
+    @menu_list = {:steak => 15, :chips => 5, :pizza =>7,
+    :lasagna =>12, :chicken => 10, :sushi => 14}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)